eduiddashboard/views/security.py

```diff
@@ -144,8 +144,8 @@ def get_authn_info(request):
     authninfo = []
 
     for credential in user.passwords.to_list_of_dicts():
-        auth_entry = request.authninfodb.authn_info.find_one({'_id': ObjectId(credential['id'])})
-        log.debug("get_authn_info {!s}: cred id: {!r} auth entry: {!r}".format(user, credential['id'], auth_entry))
+        auth_entry = request.authninfodb.authn_info.find_one({'_id': ObjectId(credential['credential_id'])})
+        log.debug("get_authn_info {!s}: cred id: {!r} auth entry: {!r}".format(user, credential['credential_id'], auth_entry))
```

Comment:
Change to use user.credentials instead of user.passwords (.passwords is deprecated).

Change to use to_list() and use credential.credential_id instead of relying on the current in-database format. Also, looks like it should probably filter on passwords.